app/templates/teachers/starter-licenses-view.jade

```diff
@@ -0,0 +1,143 @@
+        h5(data-i18n='special_offer.low_price_title', data-i18n-options=JSON.stringify({ starterLicensePrice: '$' + view.state.get('dollarsPerStudent') }))
+        p.small(data-i18n="special_offer.low_price_description", data-i18n-options=JSON.stringify(view.i18nData))
+  
+    .row.text-center.tall-margin-top
+      h2
+        span(data-i18n="special_offer.three_great_courses")
+          | Three great courses included in the Starter License:
+    .row.tall-margin-bottom.m-t-2
+      .col-xs-4
+        .text-center
+          img(src="http://localhost:3000/images/pages/home/computer-science-2.png" width=147 height=147)
+        b(data-i18n="teacher.cs2")
+        =' '
+        span(data-i18n="special_offer.cs2_description")
+      .col-xs-4
+        .text-center
+          img(src="http://localhost:3000/images/pages/home/web-development-1.png" width=147 height=147)
+        b(data-i18n="special_offer.wd1")
+        =' '
+        span(data-i18n="special_offer.wd1_description")
+        //- div
+        //-   a(href="TODO")
+        //-     span(data-i18n="special_offer.see_an_example_project")
+      .col-xs-4
+        .text-center
+          img(src="http://localhost:3000/images/pages/home/game-development-1.png" width=147 height=147)
+        b(data-i18n="special_offer.gd1")
+        =' '
+        span(data-i18n="special_offer.gd1_description")
+        //- div
+        //-   a(href="TODO")
+        //-     span(data-i18n="special_offer.see_an_example_project")
+
+    .row.text-center.tall-margin
+      h2.m-b-2
+        span(data-i18n="special_offer.get_started_today")
+      a.purchase-btn.btn.btn-lg.btn-navy
+        span(data-i18n="special_offer.purchase_starter_licenses")
+      .m-t-3
+        span(data-i18n="special_offer.want_all_the_courses")
+      a.contact-us-btn.btn.btn-forest
+        span(data-i18n="general.contact_us")
+          
+    .row.text-center.tall-margin
+      h2.m-b-3
+        span(data-i18n="special_offer.compare_license_types")
+
+      table.feature-comparison-table
+        tr
+          th
+          th
+          th.special-offer-col
+            i
+              b Special Offer
+          th
+        tr
+          th
+          th(data-i18n="teacher.trial")
+          th.special-offer-col(data-i18n="teacher.starter_license")
+          th(data-i18n="teacher.full_license")
+        tr
+          th(data-i18n="special_offer.cs")
+          td
+            span(data-i18n="courses.course")
+            =" 1"
+          td.special-offer-col
+            span.text-capitalize(data-i18n="courses.courses")
+            =" 1–2"
+          td
+            span.text-capitalize(data-i18n="courses.courses")
+            =" 1–5+"
+        tr
+          th(data-i18n="special_offer.gd")
+          td x
+          td.special-offer-col
+            span(data-i18n="courses.course")
+            =" 1"
+          td
+            span.text-capitalize(data-i18n="courses.courses")
+            =" 1–2+"
```

Comment:
I'm not sure these will work in translation. Better to have specific strings to translate like "Courses 1-2".